src/main/java/com/microsoft/build/YmlFilesBuilder.java

```diff
@@ -166,7 +167,7 @@ void addChildren(TypeElement classElement, List<String> children) {
 
     List<? extends Element> filterPrivateElements(List<? extends Element> elements) {
         return elements.stream()
-                .filter(element -> !ElementUtil.isPrivateOrPackagePrivate(element)).collect(Collectors.toList());
+                .filter(element -> !Utils.isPrivateOrPackagePrivate(element)).collect(Collectors.toList());

```

Comment:
Naming, be more specified about what util.

Reply:
Reused the code from java standard doclet source code, so followed its' class naming:

http://hg.openjdk.java.net/jdk/jdk11/file/1ddf9a99e4ad/src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/Utils.java